core/Piranha.Azure.BlobStorage/BlobStorageExtensions.cs

```diff
@@ -62,21 +42,24 @@ public static class BlobStorageExtensions
     /// </summary>
     /// <param name="services">The current service collection</param>
     /// <param name="credentials">The auth credentials</param>
-    /// <param name="containerName">The optional container name</param>
+    /// <param name="blobContainerUri">
+    /// A <see cref="Uri"/> referencing the blob service.
+    /// This is likely to be similar to "https://{account_name}.blob.core.windows.net".
+    /// </param>
     /// <param name="naming">How uploaded media files should be named</param>
     /// <param name="scope">The optional service scope. Default is singleton</param>
     /// <returns>The service collection</returns>
     public static IServiceCollection AddPiranhaBlobStorage(
         this IServiceCollection services,
-        StorageCredentials credentials,
```

Comment:
This constitutes the main breaking change when moving to the v.12 Azure SDK, as the `StorageCredentials` class is no longer available.